pkg/core/src/graphics/Camera.ts

```diff
@@ -1,14 +1,23 @@
-import { Renderable, RenderableProps } from "@piggo-legends/core";
+import { Renderable, RenderableProps } from "@piggo-legends/contrib";
+import { Renderer } from "@piggo-legends/core";
+import { Container } from "pixi.js";
 
-export class Camera extends Renderable<RenderableProps> {
```

Comment:
no longer a `Renderable` because `Renderable` is a `Component`

`Camera` needs to live outside of the ECS context